src/styles/components/_sandpack.scss

```diff
@@ -0,0 +1,163 @@
+// the article container
+.sandpack {
+  header {
+    color: var(--foreground-color);
+
+    h2 {
+      position: relative;
+      font-size: 1.5625rem; // 25px/16
+      font-family: Viga, sans-serif;
+      line-height: 2.125rem; // 34px/16
+      margin-inline: auto;
+
+      &::after {
+        content: "";
+        position: absolute;
+        width: 2rem;
+        height: 2.5rem;
+        background-image: url("/images/cmt_inventory_arrow.svg");
+        transform: translateX(0.5rem) rotate(-30deg);
+        background-repeat: no-repeat;
+        background-position: center;
+        background-size: cover;
+
+        @media (prefers-color-scheme: dark) {
+          filter: invert(1);
+        }
+      }
+
+      @media (min-width: $breakpoint-increase-content-font-size) {
+        font-size: map.get($font-sizes, "md");
+        line-height: 2.5rem; // 40px/16
```

Comment:
`line-height` should be unitless 🙃

Reply:
I did this ALL OVER!!! 🤦‍♀️

Reply:
They're fixed: https://github.com/sparkbox/accessible-components/pull/105/commits/1fa508dd12b7bc45b33ba214d1f04fd1ea5dae89